README.md

```diff
@@ -1,2 +1,316 @@
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="images/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+<!--
+  <h3><b>Microverse README Template</b></h3>
+-->
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# <i class='bx bxs-book bx-spin' style='color:#5e3b03' ></i> Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# <i class='bx bxs-package bx-fade-up' style='color:#927538' ></i> PERUVIAN SOFTWARE CONFERENCE 2023 <a id="about-project"></a>
+
+This repository is for a fictitious conference on my country. This is the final assignment of Module One named Capstone Project.
+
+The **Peruvian Software Conference** (formerly Peruvian Science and Information Conference) is a research conference held in Paracas, Perú since 2013. The annual conference series included keynotes, special sessions, poster presentations, tutorials, workshops and papers.
+
+## 🛠 Built With <a id="built-with"></a>
+
+### Tech Stack <a id="tech-stack"></a>
+
+This project is coded with standard languges like HTML, CSS and JavaScript.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/multipage/">HTML 5 <i class='bx bxl-html5 bx-tada' style='color:orange' ></i></a></li>
+    <li><a href="https://html.spec.whatwg.org/multipage/">CSS 3 <i class='bx bxl-css3 bx-tada' style='color:#23a1de'  ></i></a></li>
+    <li><a href="https://html.spec.whatwg.org/multipage/">JavaScript <i class='bx bxl-javascript bx-tada' style='color:#fbee36'  ></i></a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a id="key-features"></a>
+
+In this project we implement:
+
+- **Fetch JSON data.** We retrieve data from a JSON file to dynamically create HTML for the speaker’s section of the page.
+- **Responsiveness.** We use media queries, DOM events and listeners, and mobile first approach to make our page responsive.
+- **Semantic HTML** We use semantic HTML tags to improve the accessibility of our page.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Video Presentation ##
+
+- [Vimeo Link](https://vimeo.com/775149846)
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a id="live-demo"></a>
+
+To see a Live Demo Version of our page, you kindly follow the next link:
+
+- [Live Demo Link](https://elafildecolor.github.io/PSC2023-Capstone/)
```

Comment:
- [ ] Good job on your README, it looks great!! However, this Live Demo link is not working. Please work on it again to make sure that it is working 😃 👍